MApp.js

```diff
@@ -0,0 +1,110 @@
+document.addEventListener('DOMContentLoaded', () => {
+   fetch('https://ghibliapi.herokuapp.com/films') //we want the data here before user interacton
+   .then(response => response.json())
+
+   .then(dataObj => {
+      fetchMovie(dataObj)
+
+      // fetchMovie(response)
+   })
+
+   // .catch(err =>{
+   //    console.log("Nah Dawg", err)
+   // })
+
+   eventListeners()
+})
+
+var iDontBelong = [null] //API response 
+ //common endpoint
+function fetchMovie(film){  //code initially existed at the end
+   let button3 = document.querySelector('#sumnToWatch')
+   for(let i of film){
+   let filmData = document.createElement('option')
+      filmData.innerText = i.title
+
+      button3.appendChild(filmData)
+
+      iDontBelong.push(i.id)
+      }
+   }
+
+   //No idea on how to add default clear empty option
+const eventListeners = () => {
+   // let button = document.querySelector('#ama')
+   //    button.addEventListener('click', fetchMovie)
+
+   let button2 = document.querySelector('#submit')
+      button2.addEventListener('click', postReview)
+   
+   let button3 = document.querySelector('#sumnToWatch')
+      button3.addEventListener('change', (myFunctionName) => {
+         let empDiv = document.querySelector('#ugh')
+         let long = iDontBelong[myFunctionName.target.selectedIndex] //https://developer.mozilla.org/en-US/docs/Web/API/HTMLSelectElement/selectedIndex
```

Comment:
Good way of getting id for the movie. Later on when you learn about React, you will be able to make one fetch request and save that data from the request to use it multiple times without making new fetch requests everytime the user chooses new movie.